test/integration/services/Issues.ts

```diff
@@ -19,12 +19,40 @@ beforeEach(() => {
 
 describe('Issues.create', () => {
   it('should create a valid issue on a project', async () => {
-    const issue = await service.create(project.id, {
-      title: 'Issue Integration test',
+    const issue1 = await service.create(project.id, {
+      title: 'Issue Integration test1',
       description: 'A test issue ensuring a sucessfully create Issue in GitLab',
     });
 
-    expect(issue).toBeInstanceOf(Object);
-    expect(issue.title).toBe('Issue Integration test');
+    const issue2 = await service.create(project.id, {
+      title: 'Issue Integration test2',
+      description: 'A test issue ensuring a sucessfully create Issue in GitLab',
+    });
+
+    expect(issue1).toBeInstanceOf(Object);
+    expect(issue1.title).toBe('Issue Integration test1');
   });
 });
+
+describe('Issues.all', () => {
+  it('should return a list of issues on a project', async () => {
+    const issues = await service.all(project.id);
+
+    expect(issues).toBeInstanceOf(Array);
+    expect(issues.length).toEqual(2)
```

Comment:
we should look into `lint-staged` and apply `tslint -fix`

Reply:
Added, give it a look to make sure i didnt mess up the configuration